cfgov/unprocessed/apps/paying-for-college/css/college-costs/financial-item.less

```diff
@@ -65,22 +72,25 @@
     }
   }
 
-  &.o-expandable_header {
-    &[aria-expanded='false'] {
-      .o-expandable_cue-close {
+  &.o-costs-group_target {
```

Comment:
`o-expandable_target` is gone, as it was redundant to the header. We should probably follow suit here and have it be `o-costs-group_header`.

Reply:
Sounds good, I'll work that in.